docs/getting_started/tutorial-hyperbolic-discounting.ipynb

```diff
@@ -0,0 +1,1402 @@
+{
```

Comment:
<p>You can make the code for the figure a lot denser by looping over each series with<code>for i, series in enumerate([df, df_beta, df_lowbeta])</code> and using the running variable to adjust the width of the bars and select the correct colors.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/OpenSourceEconomics/respy/pull/347/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='19'/>